docs/reference/form_types.rst

```diff
@@ -357,6 +357,10 @@ The available options are:
 ``dropdown_item_css_class``
   defaults to "". CSS class of dropdown item.
 
+``safe_label``
+  defaults to ``false``. Set to ``true`` to enable the label to be displayed as Raw HTML,
+  which may cause XSS vulnerability
```

Comment:
please add a dot at the end

Reply:
already done with commit 8e89a36e8836e62b8e51f86a37683c75e4cfbcd2